src/pages/home.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Figure from 'react-bootstrap/Figure'
+
+import CURRENT_USER from '../apollo/queries/currentUser'
+import { useQuery } from '@apollo/react-hooks'
+
+function Home (props){
+    let firstName,lastName = ""
+    
+    const{ loading, error, data }= useQuery(CURRENT_USER)
+    if (!loading & !error){
+        console.log(loading, error, data)
+        firstName= data.currentUser.firstName
+        lastName= data.currentUser.lastName
+
+    }
```

Comment:
I suggest this here:

```suggestion
function Home (){    
    const { loading, error, currentUser } = useCurrentUserQuery() // This should be a new query hook
    
    if (loading) return null
    if (error) return 'Something went wrong'
    
    // Then below just use {currentUser.firstName} {currentUser.lastName}
```